app/routes/datamart/land.py

```diff
@@ -0,0 +1,171 @@
+"""Run analysis on registered datasets."""
+
+import random
+import uuid
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, Path, Query, Request
+from fastapi.logger import logger
+from fastapi.openapi.models import APIKey
+from fastapi.responses import ORJSONResponse
+
+from app.models.pydantic.datamart import TreeCoverLossByDriverIn
+
+from ...authentication.api_keys import get_api_key
+from ...models.pydantic.responses import Response
+
+router = APIRouter()
+
+
+@router.get(
+    "/tree-cover-loss-by-driver",
+    response_class=ORJSONResponse,
+    response_model=Response,
+    tags=["Land"],
+)
+async def tree_cover_loss_by_driver_search(
+    *,
+    geostore_id: UUID = Query(..., title="Geostore ID"),
+    canopy_cover: int = Query(..., alias="canopy_cover", title="Canopy Cover Percent"),
```

Comment:
Should we have a default here? Don't we default elsewhere to 30 or 35 percent I think?